superset/migrations/alembic.ini

```diff
@@ -25,7 +25,7 @@ handlers = console
 qualname =
 
 [logger_sqlalchemy]
-level = WARN
```

Comment:
I like having more insight into what alembic is doing. I think that level of verbosity is a good idea for such risky operations like database upgrading.